src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+* {
+  margin: 0;
+}
```

Comment:
Don't use `*` selector (it impacts performance). It's better to set styles only for elements that require them. Consider removing the universal selector and applying the margin reset to specific elements or using a more targeted reset approach.